### PR TITLE
Dat 2738 introduce aside tag into parser

### PR DESCRIPTION
**Pull request by idradm**
@macbre @SebastianMarzjan @jacekjursza

### CODE REVIEW

**Comment by macbre** [comment]
Worth adding a unit test here? :)

File: extensions/wikia/PortableInfobox/PortableInfoboxHooks.class.php [hunk 1]
```diff
@@ -1,8 +1,9 @@
 <?php
+
 class PortableInfoboxHooks {
 	// FIX ME: temporary implementation - styles should be included only on page with portable infobox
```
Comment (SebastianMarzjan) [inline extensions/wikia/PortableInfobox/PortableInfoboxHooks.class.php]: Not related to this commit, but should be changed to TODO: Add to global css on sitewide release

Comment (macbre) [inline extensions/wikia/PortableInfobox/PortableInfoboxHooks.class.php]: Or consider using ResourceLoader and registering a module with Infoboxes CSS as a dependency.

**Comment by SebastianMarzjan** [comment]
Formatting seems off in the test, logic LGTM. Thanks for acting on feedback! :)

**Pull request merged**